ccd-definition/CaseField/CaseFieldLRspec.json

```diff
@@ -762,5 +762,20 @@
     "HintText": "For example 9 December 2020",
     "FieldType": "RespondToClaimAdmitPartLRspec",
     "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "respondent1CourtOrderPaymentByPlan",
+    "Label": " ",
+    "FieldType": "PayingMoneyAsAResultOfCourtOrder",
```

Comment:
Can we rename to PaymentUponCourtOrder

Reply:
or rename to CourtOrderPayment

Reply:
Bhabani, I think case fieltype name is okay, and its not a big name either, can we resolve it ?